src/components/Calculator.js

```diff
@@ -0,0 +1,32 @@
+import '../styles.css';
+
+function CalculatorApp() {
```

Comment:
- [ ] Kindly change your Calculator component to be class-based, this is a project requirement.